### ISSUE TITLE
Datepicker input name attribute

### ISSUE THREAD

**Issue by Stetzon** [wffranco/vue-strap#64]
Hi, would it be possible to add a **name** attribute to the Datepicker input element?  This attribute is already present on the other input components (checkbox, radio, input, etc).  Having trouble POSTing form data from the datepicker without it.

`   
 <input class="form-control datepicker-input" type="text"
      v-model="val"
      :class="{'with-reset-button': clearButton}"
      :placeholder="placeholder"
      :name="name"  
      :style="{width:width}"
      @click="inputClick"
 />
`

**Comment by Stetzon** [comment]
#66 Submitted pull request

**Comment by jeitnier** [comment]
If I import the component like this:

    import { datepicker } from 'vue-strap'

It puts the `name` attribute on the parent `div`.

**Comment by leocxy** [comment]
@jeitnier 
Yes. Your right. But I am confused. Because I checkd the Datepicker.vue. I found the name attribute on the input...
`<template>
  <div class="datepicker">
    <input class="form-control datepicker-input" type="text"
      v-model="val"
      :class="{'with-reset-button': clearButton}"
      :name="name"
      :placeholder="placeholder"
      :style="{width:width}"
      @click="inputClick"
    />
    <button v-if="clearButton&&val" type="button" class="close" @click="val = ''">
      <span>&times;</span>
    </button>
    <div class="datepicker-popup" v-show="displayDayView">
      <div class="datepicker-inner">
        <div class="datepicker-body">
          <div class="datepicker-ctrl">
            <span :class="preBtnClasses" aria-hidden="true" @click="preNextMonthClick(0)"></span>
            <span :class="nextBtnClasses" aria-hidden="true" @click="preNextMonthClick(1)"></span>
            <p @click="switchMonthView">{{stringifyDayHeader(currDate)}}</p>
          </div>
          <div class="datepicker-weekRange">
            <span v-for="w in text.daysOfWeek">{{w}}</span>
          </div>`

**Comment by leocxy** [comment]
@Stetzon 
Hello Stetzon, I got the same issues with you.
I fixed this issues with import vue-strap with real path.

`
#Before
import {datepicker} from 'vue-strap'
#After
import { modal, navbar, dropdown, input, select, radio, buttonGroup, alert, datepicker } from '../../../../node_modules/vue-strap';
`
